tex_inspection/tex_inspection/tex_inspection.py

```diff
@@ -293,38 +331,61 @@ def fetch_00readme_v2(self, filename: str) -> None:
         """Read and parse 00README.XXX file, v2"""
         stem, ext = os.path.splitext(filename)
         zzrm = None
-        with open(filename, "rb") as fd:
-            match ext:
-                case ".yml" | ".yaml":
-                    loader = YAML()
+        match ext:
+            case ".yml" | ".yaml":
+                loader = YAML()
+                with open(filename, "rb") as fd:
                     zzrm = loader.load(fd)
-                case ".json" | ".jsn" | ".ndjson":
+            case ".json" | ".jsn" | ".ndjson":
+                with open(filename, "rb") as fd:
                     zzrm = json.load(fd)
-                case ".toml":
-                    zzrm = tomli.load(fd)
+            case ".toml":
+                zzrm = toml.load(filename)
```

Comment:
Do we document somewhere the order extensions are tested?
Users might get lost having a `00README.yaml` and `00README.json` and `00README.....`